src/patent_client/epo/ops/register/api.py

```diff
@@ -0,0 +1,175 @@
+import logging
```

Comment:
Let's clean this file up, and get rid of the API's we don't use. Register only has a Search / Retrieval / Events / Procedural Steps endpoints, so we can delete the rest.

Remember, this is all in version control, so don't worry about losing information by deleting it, it will always be in the version history!